augraphy/augmentations/markup.py

```diff
@@ -166,6 +167,14 @@ def draw_line(self, p1, p2, markup_mask, markup_thickness, reverse):
     def __call__(self, image, layer=None, force=False):
         markup_img = image.copy()
         overlay = markup_img.copy()
+
+        if self.markup_color == "random":
+            self.markup_color = (random.randint(0, 255), random.randint(0, 255), random.randint(0, 255))
```

Comment:
In this edit, i added section to generate random colour and the inverted colour (contrast with background). 

For markup, i think it will be more relevant to have it in ink phase, so in that case, most of the time we will not able to detect the colour from paper phase and generate a colour with higher contrast.